bucket/oripa.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "0.36",
+    "homepage": "http://mitani.cs.tsukuba.ac.jp/oripa/",
+    "description": "Drawing software dedicated to designing the crease patterns of origami",
+    "license": "GPL-3.0-or-later",
+    "url": "https://github.com/oripa/oripa/releases/download/v0.36/oripa-0.36.jar#/oripa.jar",
+    "hash": "5e72f6aea5722e80a11fa8098cf3cd1f9c21be67014da39360abf640be194ebc",
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\oripa.jar\" -Switches 'icon\\oripa.gif'",
+        "Set-Content \"$dir\\oripa.bat\" \"start javaw -jar %~dp0\\$fname %*\" -Encoding ASCII",
+        "$stream = [System.IO.File]::OpenWrite(\"$dir\\icon.ico\")",
```

Comment:
**This is for Windows Powershell 5.1 (Native)**
Apparently you need to grab the type before using `[System.Drawing]`

```suggestion
        "Add-Type -AssemblyName System.Drawing",
        "$stream = [System.IO.File]::OpenWrite(\"$dir\\icon.ico\")",
```

Neverthless, even after this fix, for some reason the icon doesn't show up - does it relate to the fact that the source is actually not a square? (Dimensions are 30x31 pixels)

![image](https://user-images.githubusercontent.com/45321798/72560162-5d541080-389e-11ea-99f3-49ec99656253.png)